pkg/rancher-desktop/components/DashboardOpen.vue

```diff
@@ -0,0 +1,35 @@
+<script lang="ts">
+import Vue from 'vue';
+import { mapGetters } from 'vuex';
+
+import { State as K8sState } from '@pkg/backend/backend';
+
+export default Vue.extend({
+  name:     'dashboard-button',
+  computed: {
+    ...mapGetters('preferences', ['getPreferences']),
+    ...mapGetters('k8sManager', { k8sState: 'getK8sState' }),
+    kubernetesEnabled(): boolean {
+      return this.getPreferences.kubernetes.enabled;
+    },
+    kubernetesStarted(): boolean {
+      return this.k8sState === K8sState.STARTED;
+    },
+  },
+  methods: {
+    openDashboard() {
+      this.$emit('open-dashboard');
+    },
+  },
+});
+</script>
+
+<template>
+  <button
+    v-if="kubernetesEnabled && kubernetesStarted"
```

Comment:
I think the ideal behaviour is:

State | Behaviour
--- | ---
Kubernetes disabled | Button not visible
Kubernetes enabled, but still starting | Button visible, but disabled
Kubernetes running | Button visible and enabled

Does that seem like it makes sense?